Exchange-Guide.md

```diff
@@ -0,0 +1,196 @@
+  "numPeers": 3,
+  "numPairs": 1,
+  "orders": {
+    "peer": 1,
+    "own": 0
+  },
+  "lndbtc": {
+    "error": "",
+    "channels": {
+      "active": 1,
+      "inactive": 0,
+      "pending": 0
+    },
+    "chainsList": [
+      "bitcoin"
+    ],
+    "blockheight": 18797,
+    "urisList": [],
+    "version": "0.5.0-beta commit=v0.4.2-beta-1247-ge22ae9985b696924d5934004c269c930d3ccba43",
+    "alias": "BTC@exchange-install-test-machine"
+  },
+  "lndltc": {
+    "error": "",
+    "channels": {
+      "active": 1,
+      "inactive": 0,
+      "pending": 0
+    },
+    "chainsList": [
+      "litecoin"
+    ],
+    "blockheight": 22362,
+    "urisList": [],
+    "version": "0.5.0-beta commit=v0.4.2-beta-1247-ge22ae9985b696924d5934004c269c930d3ccba43",
+    "alias": "LTC@exchange-install-test-machine"
+  }
+}
+```
+
+with one active channel for both, `lndbtc` and `lndltc`. If so, you are finally ready to rumble. Yay - let's do some test trades!
+
+### Trading
+`xud` is now connected to `xud-simnet` which includes three permanent nodes operated by Exchange Union. You can view these by running
+```
+xucli listpeers
+```
+The output of connected peers should contain
+```
+"address": "xud1.test.exchangeunion.com:8885",
+"nodePubKey": "02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6",
+
+"address": "xud2.test.exchangeunion.com:8885",
+"nodePubKey": "028599d05b18c0c3f8028915a17d603416f7276c822b6b2d20e71a3502bd0f9e0a"
+
+"address": "xud3.test.exchangeunion.com:8885",
+"nodePubKey": "03fd337659e99e628d0487e4f87acf93e353db06f754dccc402f2de1b857a319d0"
+```
+You already received existing orders from the network. You can view these with
+```
+xucli getorders LTC/BTC
+```
+Let's execute a test order to trigger a match and execution via atomic swap by e.g. buying 0.0005 litecoin with bitcoin
+```
+xucli buy 0.0005 LTC/BTC 1.13
+```
+which should result in sth. like this:
+```
+matched 0.0001 BTC @ 1.13 with order da8d3401-e75a-11e8-972d-a9036be5a943
+No more matches found
+```
+
+Currently the trading bots of xud1, 2 & 3 are configured to simply replace successfully filled and swapped orders. A more realistic, market dynamic behavior will be deployed soon.
+
+### API Integration
+`xud`'s gRPC API overview can be found [here](https://exchangeunion.github.io/xud-typedoc/classes/_grpc_grpcservice_.grpcservice.html).
+
+From a high-level perspective, `xud` supports two modes:
+1. `matching`
+2. `nomatching`
+
+Default is `matching` mode, which means that `xud` acts as order book & matching engine for all of your orders. Choose this mode if your exchange is not live yet, comparably small or you simply want to rather rely on the battle-tested matching engine of `xud` instead of using your own. You will need to design your exchange in a way to add orders to `xud`'s order book and listen to order events, in order to display real-time order information to your users and update user account balances. The following API calls are relevant:
```

Comment:
"battle-tested" is a bit generous for now, one day though.

Reply:
True, just removed it.